ovs/src/main/java/org/dcsa/conformance/standards/ovs/party/OvsPublisher.java

```diff
@@ -96,15 +88,78 @@ private void supplyScenarioParameters(JsonNode actionPrompt) {
   public ConformanceResponse handleRequest(ConformanceRequest request) {
     log.info("OvsPublisher.handleRequest(%s)".formatted(request));
 
+    Map<String, List<OvsAttributeMapping>> ovsAttributeMappings =
+        OvsAttributeMapping.initializeAttributeMappings();
+
     JsonNode jsonResponseBody =
         JsonToolkit.templateFileToJsonNode(
             "/standards/ovs/messages/ovs-%s-response.json"
                 .formatted(apiVersion.toLowerCase().replaceAll("[.-]", "")),
             Map.ofEntries());
 
-    return request.createResponse(
-        200,
-        Map.of(API_VERSION, List.of(apiVersion)),
-        new ConformanceMessageBody(jsonResponseBody));
+    ArrayNode filteredArray = OBJECT_MAPPER.createArrayNode();
```

Comment:
Is this for dynamically generating the response from the static file according to the filter params?

Reply:
Yes, indeed.. The publisher is created such a way that it exhibits real behavior.